docs/backup-restore.md

```diff
@@ -1,134 +1,204 @@
 
-### Backup with explicit pausing
+However, if the quorum of cluster nodes loses their data, you will need a transactionally consistent backup to restore the cluster to a certain transactionally consistent point.
 
-One way to create a transactionally-consistent backup is to take a backup while ScalarDB cluster does not have outstanding transactions.
-If an underlying database supports a point-in-time snapshot/backup mechanism, you can take a snapshot during the period.
-If an underlying database supports a point-in-time restore/recovery mechanism, you can set a restore point to a time (preferably the mid-time) in the period.
+To create a transactionally consistent cluster-wide backup, pause the application that is using ScalarDB or ScalarDB Server and take snapshots of the nodes as described in [Back up with explicit pausing](#back-up-with-explicit-pausing) or stop the Cassandra cluster, take copies of all the data in the nodes, and start the cluster.
 
-To easily make ScalarDB drain outstanding requests and stop accepting new requests to create a pause duration, it is recommended to use [ScalarDB Server](scalardb-server.md) (which implements `scalar-admin` interface) or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your ScalarDB applications.
-With [scalar-admin client tool](https://github.com/scalar-labs/scalar-admin/tree/main/java#scalar-admin-client-tool), you can pause nodes/servers/applications that implement the scalar-admin interface without losing ongoing transactions.
+{% capture notice--info %}
+**Note**
 
-Note that when you use a point-in-time-restore/recovery mechanism, it is recommended to minimize the clock drifts between clients and servers by using clock synchronization such as NTP.
-Otherwise, the time you get as a paused duration might be too different from the time in which the pause was actually conducted, which could restore to a point where ongoing transactions exist.
-Also, it is recommended to pause a long enough time (e.g., 10 seconds) and use the mid-time of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+To avoid potential issues, you should use [Cassy](https://github.com/scalar-labs/cassy), which is a backup tool for Cassandra. Cassy is also integrated with `scalar-admin`, so the tool can issue a pause request to the Cassandra cluster of your application that is using ScalarDB or ScalarDB Server. For more details about how to create a backup by using Cassy, see [Take cluster-wide consistent backups](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups).
```

Comment:
Should we change `ScalarDB Server` to `ScalarDB Cluster` here?

```suggestion
To avoid potential issues, you should use [Cassy](https://github.com/scalar-labs/cassy), which is a backup tool for Cassandra. Cassy is also integrated with `scalar-admin`, so the tool can issue a pause request to the Cassandra cluster of your application that is using ScalarDB or ScalarDB Cluster. For more details about how to create a backup by using Cassy, see [Take cluster-wide consistent backups](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups).
```